.travis.yml

```diff
@@ -28,6 +28,10 @@ matrix:
       env: FS_BACKEND=OFF DOCKERFILE=ub1804
 
 addons:
+  apt:
+    update: true
+    packages:
+      - lcov
```

Comment:
That is in theory not needed anymore, right @achilleas-k

Reply:
Correct.  Remove it.

Reply:
No U!